src/BugsnagUnityForReview.mm

```diff
@@ -0,0 +1,876 @@
+#import "Bugsnag.h"
+#import "BugsnagConfiguration+Private.h"
+#import "BugsnagLogger.h"
+#import "BugsnagUser.h"
+#import "BugsnagNotifier.h"
+#import "BugsnagSessionTracker.h"
+#import "Bugsnag+Private.h"
+#import "BugsnagClient+Private.h"
+#import "BSG_KSSystemInfo.h"
+#import "BSG_KSMach.h"
+#import "BSG_KSCrash.h"
+#import "BSG_RFC3339DateTool.h"
+#import "BugsnagBreadcrumb+Private.h"
+#import "BugsnagSession+Private.h"
+
+extern "C" {
```

Comment:
If we make this a `.m` instead of `.mm` file, we can remove the whole `extern "C"` section (we'll need to keep the definition of `struct bugsnag_user` though)

Reply:
done and seems to work great 👍